flowmachine/tests/test_to_sql.py

```diff
@@ -28,29 +28,20 @@ def test_stores_table(flowmachine_connect):
     EventTableSubset().to_sql() can be stored as a TABLE.
     """
     query = EventTableSubset("2016-01-01", "2016-01-01 01:00:00")
-    query.to_sql(schema="tests", name="test_table")
+    query.to_sql(name="test_table", schema="tests").result()
     assert "test_table" in flowmachine_connect.inspector.get_table_names(schema="tests")
 
 
-def test_stores_view(flowmachine_connect):
-    """
-    EventTableSubset().to_sql() can be stored as a VIEW.
-    """
-    query = EventTableSubset("2016-01-01", "2016-01-01 01:00:00")
-    query.to_sql(schema="tests", name="test_view", as_view=True)
-    assert "test_view" in flowmachine_connect.inspector.get_view_names(schema="tests")
-
-
 def test_can_force_rewrite(flowmachine_connect, get_length):
     """
     Test that we can force the rewrite of a test to the database.
     """
     query = EventTableSubset("2016-01-01", "2016-01-01 01:00:00")
-    query.to_sql(schema="tests", name="test_rewrite")
+    query.to_sql(name="test_rewrite", schema="tests").result()
```

Comment:
I was wondering why we need the extra `.result()`, but then I realised it's because `to_sql()` now returns a future.

When I looked up what `to_sql()` actually does it says: "Store the result of the calculation back into the database." To be honest, I find this a little confusing. Either the name is wrong (`to_sql()` sounds to me like it should simply return the SQL string of a Query object) or its primary purpose is something different.

Either way (and also looking at the implementation in `query.py`), it feels like this needs some clarification and tidying up. Not for this PR, but I wanted to note it here so I don't forget.

Reply:
I believe it was because of this: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_sql.html

Reply:
Oh I see, yes that could well be the reason. Do you feel the name is adequate? My personal impression is that our interface & use case is sufficiently different from pandas now that the name is more confusing than it helps.